### PR TITLE
Sc 18

### PR DESCRIPTION
**Pull request by alanahodge**

### CODE REVIEW

File: src/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    fontSize: "40px",
+    paddingTop: "5px",
+    display: "flex",
+    alignItems: "center",
+    flexDirection: "column",
+  },
+  
+})
+
+//class LoggingButton extends React.Component{
```
Comment (NavyaGupta) [inline src/HomePage.js]: Remove?

File: src/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <div className={classes.body}>
+        <Button className={classes.PlayButton}>
+          Start Playing
+        </Button>
+        
+        <h2>The best place to stay connected</h2>
+      </div>
+
+      <div style={{ width: '100%' }}>
+      <Box display="flex" p={1} flexDirection="row" justifyContent="center" >
```
Comment (NavyaGupta) [inline src/HomePage.js]:
```suggestion
      {className=classes.box}
```
Make this a style class
```box: {display="flex" p={1} flexDirection="row" justifyContent="center"} ```

File: src/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+import { TopHeader } from "./shared/TopHeader";
+import chessBoard from "./hostGamePage/chessBoard.png";
+import Box from '@material-ui/core/Box';
+import { flexbox } from '@material-ui/system';
+
+
+const styles = makeStyles({
+  backButton: {
+    marginTop: "10px",
+  },
```
Comment (NerajM) [inline src/HomePage.js]: Not being used anywhere, not needed

File: src/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        <Button className={classes.PlayButton}>
+          Start Playing
+        </Button>
+        
+        <h2>The best place to stay connected</h2>
+      </div>
+
+      <div style={{ width: '100%' }}>
+      <Box display="flex" p={1} flexDirection="row" justifyContent="center" >
+        <Box p={1} paddingRight = "100px">
```
Comment (NavyaGupta) [inline src/HomePage.js]: Same with this

File: src/HomePage.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+          <h2>Connect With Friends or Strangers</h2>
+        </Box>
+        <Box p={1} paddingRight="100px">
```
Comment (NavyaGupta) [inline src/HomePage.js]: Same here

File: src/HomePage.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+          <h2>Communicate via Chat</h2>
+        </Box>
+        <Box p={1} paddingRight="100px">
```
Comment (NavyaGupta) [inline src/HomePage.js]: Same here. All this `paddingRight` could be in one class

File: src/HomePage.js (hunk 2 above)
Comment (NerajM) [inline src/HomePage.js:65]: Put into styling

File: src/HomePage.js [hunk 7]
```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { Button, makeStyles, useEventCallback} from "@material-ui/core";
```
Comment (NavyaGupta) [inline src/HomePage.js]: Not using `useEventCallaback`

File: src/HomePage.js [hunk 8]
```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
```
Comment (NavyaGupta) [inline src/HomePage.js]: Not using `useState`

File: src/HomePage.js [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+import { Button, makeStyles, useEventCallback} from "@material-ui/core";
+import { TopHeader } from "./shared/TopHeader";
+import chessBoard from "./hostGamePage/chessBoard.png";
+import Box from '@material-ui/core/Box';
+import { flexbox } from '@material-ui/system';
```
Comment (NavyaGupta) [inline src/HomePage.js]: Dont need `flexbox`

**Review by NavyaGupta — APPROVED** [review]
Looks amazing!! 😍

**Pull request merged**